src/Components/LoginForm/index.jsx

```diff
@@ -0,0 +1,152 @@
+      setFormData((prevState) => ({
+        ...prevState,
+        passwordField: {
+          ...prevState.passwordField,
+          fieldValue: value
+        },
+        invalidCredentialsError: false
+      }));
+
+    };
+
+    const handleNameValidation = (value) => {
+      const {
+        correct,
+        empty,
+        valid
+      } = validateInput(value, 'name');
+
+      setFormData(prevState => ({
+        ...prevState,
+        nameFiled: {
+          ...prevState.nameFiled,
+          fieldValue: value,
+          isFieldTouched: true,
+          isFieldCorrect: correct,
+          isFieldEmpty: empty,
+          isFieldPassValidation: valid
+        }
+      }));
+    };
+
+    const handlePasswordValidation = (value) => {
+      const {
+        correct,
+        empty,
+        valid
+      } = validateInput(value, 'password');
+      setFormData(prevState => ({
+        ...prevState,
+        passwordField: {
+          ...prevState.passwordField,
+          fieldValue: value,
+          isFieldTouched: true,
+          isFieldCorrect: correct,
+          isFieldEmpty: empty,
+          isFieldPassValidation: valid
+        }
+      }));
+    };
+
+    const handleFormSubmit = (e) => {
+      e.preventDefault();
+      if (formData.nameFiled.isFieldCorrect && formData.passwordField.isFieldCorrect) {
+        return setAuthStatus(!isUserAuthorized);
+      }
+      setFormData(prevState => ({
+        ...prevState,
+        passwordField: {
+          ...prevState.passwordField,
+          fieldValue: '',
+          isFieldTouched: false
+        },
+        invalidCredentialsError: true
+      }));
+    };
+
+    const isButtonDisabled = () => {
+      const nameFieldCheck = !formData.nameFiled.isFieldEmpty && formData.nameFiled.isFieldTouched;
+      const passwordFieldCheck = !formData.passwordField.isFieldEmpty && formData.passwordField.isFieldTouched;
+      return !(nameFieldCheck && passwordFieldCheck);
+    };
+
+    const { nameFiled, passwordField, invalidCredentialsError } = formData;
+
+    return (
+      <div className={styles.FormWrapper}>
+        {invalidCredentialsError && <div className={styles.ErrorBanner}>
+          <ErrorBanner errorText="Wrong login or Password" />
+        </div>
+        }
+        <form className={styles.Form}>
+          <div>
+            <div className={styles.FormInput}>
+              <AuthInput
+                inputName='Login'
+                onChangeHandler={handleLoginChange}
+                blurEventHandler={handleNameValidation}
+                inputData={nameFiled}
+              />
+              <AuthInput
+                inputType='password'
+                inputName='Password'
+                onChangeHandler={handlePasswordChange}
+                blurEventHandler={handlePasswordValidation}
+                inputData={passwordField}
+              />
+            </div>
+            <Button
+              isDisabled={isButtonDisabled()}
```

Comment:
please convert `isSubmitDisabled` to a `boolean` constant, function does nothing here